OpenRastaSwagger/Discovery/Heuristics/DiscoverHttpMethodVerbs.cs

```diff
@@ -23,6 +24,20 @@ public bool Discover(MethodInfo publicMethod, OperationMetadata methodMetdata)
             if (methodAttribute != null)
             {
                 methodMetdata.HttpVerb = methodAttribute.Method;
```

Comment:
if it's the wrong method to match as per below, then methodMetadata was mutated even though it was passed in? I may misunderstand how that class is called, it just seems counter intuitive